src/frontends/pytorch/src/op/nms.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "openvino/opsets/opset9.hpp"
```

Comment:
No need to use both opset8 and opset9. Please just use opset9.

Reply:
done